### ISSUE TITLE
Analyzer zoom/scaling small screen cutoff

### ISSUE THREAD

**Issue by LekeFly** [betaflight/blackbox-log-viewer#524]
<img width="1552" alt="Screenshot 2021-08-08 at 22 37 52" src="https://user-images.githubusercontent.com/219316/128645027-1799fea3-94da-4a12-98f4-fcf70aeff7f3.png">

I think the screenshot speaks for it self. Most of the information needed is here, but i would like see more to the right.

**Comment by mikeller** [comment]
@LekeFly: No, it does not. Can you describe a bit better what change you would like to see?

**Comment by LekeFly** [comment]
@mikeller My bad. Was supposed to write **to the right** my bad. And maybe the screenshot isn't the best either. 

<img width="1065" alt="Screenshot 2021-08-10 at 13 58 26" src="https://user-images.githubusercontent.com/219316/128863002-7d65292f-df15-4558-8a1f-f73cd6ad2f85.png">

There's more going on up in the right corner here, but i can't zoom/scale/scroll. I see on YouTube videos etc where they got higher resolution screens that it gives a better overview.

**Comment by mikeller** [comment]
@LekeFly: So if I understand you right you'd like a way to zoom into the right side of the graph, instead of into the left side?

**Comment by LekeFly** [comment]
![38D14D91-7BA8-486E-910C-504B7AC5E98F](https://user-images.githubusercontent.com/219316/129270750-52563a93-d36c-4383-a0cf-026c8f9134b7.png)

Finally found an example. Yes like this @mikeller. I’m sorry for the unclear explanation

**Comment by LekeFly** [comment]
Not sure why this says i closed the issue.

**Comment by JoeriCapens** [comment]
It just looks like you need to log at a higher rate. To get results up to 1 kHz, you need to log at 2 kHz because of the Nyquist theorem.
